packages/core/src/domain/session/sessionState.ts

```diff
@@ -1,22 +1,52 @@
 import { isEmptyObject } from '../../tools/utils/objectUtils'
 import { objectEntries } from '../../tools/utils/polyfills'
 import { dateNow } from '../../tools/utils/timeUtils'
-import { SESSION_EXPIRATION_DELAY } from './sessionConstants'
+import { SESSION_EXPIRATION_DELAY, SESSION_TIME_OUT_DELAY } from './sessionConstants'
 
-const SESSION_ENTRY_REGEXP = /^([a-z]+)=([a-z0-9-]+)$/
+const SESSION_ENTRY_REGEXP = /^([a-zA-Z]+)=([a-z0-9-]+)$/
 const SESSION_ENTRY_SEPARATOR = '&'
 
+export const enum SessionExpiredReason {
+  UNKNOWN = '1',
+}
+
 export interface SessionState {
   id?: string
   created?: string
   expire?: string
   lock?: string
+  isExpired?: SessionExpiredReason
 
   [key: string]: string | undefined
 }
 
+export function getInitialSessionState({ lock }: SessionState = {}): SessionState {
+  const session: SessionState = {
+    isExpired: SessionExpiredReason.UNKNOWN,
+  }
+
+  if (lock) {
+    session.lock = lock
+  }
+
+  return session
+}
+
+export function isSessionStarted(session: SessionState) {
+  return !(isEmptyObject(session) || (Object.keys(session).length === 1 && 'lock' in session))
+}
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏ This methods need to take `lock` into account because it is leaked to the session state in the `process()` method in [sessionStoreOperations.ts](https://github.com/DataDog/browser-sdk/blob/d0e13f936569bfeb68d6e697baf22ba87b3c828e/packages/core/src/domain/session/sessionStoreOperations.ts#L45)

This is a temporary solution in order to avoid larger changes in this PR.
Ultimately I would like to fix sessionStoreOperations to not leak the `lock` property